src/v2/Apps/Artwork/__tests__/ArtworkApp.jest.tsx

```diff
@@ -0,0 +1,65 @@
+import { graphql } from "react-relay"
+import { ArtworkApp_Test_Query } from "v2/__generated__/ArtworkApp_Test_Query.graphql"
+import { setupTestWrapper } from "v2/DevTools/setupTestWrapper"
+import { ArtworkAppFragmentContainer } from "../ArtworkApp"
+import { MockBoot } from "v2/DevTools"
+
+jest.unmock("react-relay")
+jest.mock("react-tracking")
+jest.mock("v2/System/Router/useRouter", () => ({
```

Comment:
This kind of looks ok, but I don't see this import in the `ArtworkApp.tsx` file at all.

https://github.com/artsy/force/blob/5249b130db6dffef039fcb2f8eb62d6c2011f10c/src/v2/Apps/Artwork/ArtworkApp.tsx#L233-L237 is what that's doing (and `useIsRouteActive` probably doesn't need to be mocked at all).

I think that can be switched use the `useRouter` hook (which should provide the `match` prop that's needed).

Reply:
I added a bunch of mocks probs related to this component being wrapped in HOCs as you said. But it works!